src/Core/Resolvers/QueryExecutor.cs

```diff
@@ -51,6 +55,10 @@ public class QueryExecutor<TConnection> : IQueryExecutor
             ConnectionStringBuilders = new Dictionary<string, DbConnectionStringBuilder>();
             ConfigProvider = configProvider;
             HttpContextAccessor = httpContextAccessor;
+            _maxResponseSizeMB = configProvider.GetConfig().MaxResponseSizeMB();
+            _maxResponseSizeLogicEnabled = configProvider.GetConfig().MaxResponseSizeLogicEnabled();
```

Comment:
considering we are looking into enabling "hot reload" for dab. It would make more sense to directly use `configProvider.GetConfig().MaxResponseSizeLogicEnabled();` when determining whether to use new code path. since this constructor is only used once.

Reply:
Fixed.